docgen/src/Styling.md

```diff
@@ -6,62 +6,174 @@ nav_groups:
 nav_sort: 1
 ---
 
+Welcome to the react-instantsearch theming tutorial!
+
+`react-instantsearch` aims to provide the best search experience your way! We believe that theming is a big part when comes the customization of the search experience. That's why react-instantsearch uses [`react-themeable`](https://github.com/markdalgleish/react-themeable) to provides easily customizable widgets. 
+
+In this tutorial, we'll cover only css modules as theming option with `react-themeable`. However, it supports much more options and you may find one that might fit your current stack better ([Check their website](https://github.com/markdalgleish/react-themeable)). 
+
+On top of that, each widgets is shipped with class names that you can use directly if you want. 
+
+In order to get you started with styling, we are going to use the example you made in the [Getting Started Tutorial](gettingstarted.html) and add some style to it. 
+If you haven't follow the [Getting Started Tutorial](gettingstarted.html) yet, please do and comeback here right after. 
+
+In this tutorial, you'll learn: 
+
+* The concept of theme in react-instantsearch.
+* How to customize a default provided theme. 
+* How to create a totally new theme for a widget.  
+
+## What's a theme in react-instantsearch 
+
 All widgets that render one or more DOM nodes accept a `theme` prop. This prop is a map of keys to corresponding `className` or `style` prop values. 
-The different theme keys supported by components are described on their respective documentation page. You can also provide themes using css-modules, Aphrodite, JSS and others if you want. 
-See [`react-themeable`](https://github.com/markdalgleish/react-themeable) for the underlying implementation and options.
 
-By default, we provide `defaultTheme` for each widgets using css-modules under the hood. Most widgets have empty default theme, but the `SearchBox`, `RangeRatings`, `Range` and `Pagination` widgets 
-are delivered with a built-in one and are ready to use. 
+The different theme keys supported by components are described on their respective documentation page. 
+
+By default, we provide `defaultClassNames` for each widgets but they are mostly not styled. However, the `SearchBox`, `RangeRatings`, `Range` and `Pagination` widgets are delivered with a built-in style and are ready to use without any configuration on your side. 
+
+>Note: default styling in `react-instantsearch` is done using css-modules. However, you have nothing to import other than the component itself to enjoy our default themes. 
```

Comment:
I was thinking that maybe for some more advanced topic, we can add them as quote style. It doesn't render nicely yet on the documentation, let me know what you think about it.